app/resources/documentation/manual/index.md

```diff
@@ -4,111 +4,59 @@
+  - Use FGD, ENT or DEF entity definition files
+  - Visualize 3D entity models in editor
 
 # Getting Started {#getting_started}
 
-This section starts off with a small introduction to the most important technical terms related to mapping for brush-based engines. Additionally, we introduce some concepts on which TrenchBroom is built. Afterwards, we introduce the welcome window, the game selection dialog, we give an overview of the main window and explain the camera navigation in the 3D and 2D views.
+First we'll begin with an overview of key concepts, then we'll tour TrenchBroom's interface and features.
 
-## Preliminaries {#preliminaries}
+> **NOTE: this is just a manual, not a tutorial.** Beyond editor functionality, this will not teach you how to make levels. See [References and Links](#references_and_links) to find a mapping community for help and support.
 
-In this section we introduce some technical terms related to mapping in Quake-engine based games. It is not important that you understand every detail of all of these terms, but in order to understand how TrenchBroom works, you should have a general idea how maps are structured and how TrenchBroom views and manages that structure. In particular, this section introduces some concepts that we added to the map structures (without changing the file format, of course). Knowing and understanding these concepts will help you to get a grip on several important aspects of editing levels in TrenchBroom.
+## Key Concepts {#key_concepts}
 
-### Map Definitions
+To use TrenchBroom effectively, you'll need to know some core Quake engine concepts and workflows.
 
-In Quake-engine based games, levels are usually called maps. The following simple specification of the structure of a map is written in extended Backus-Naur-Form (EBNF), a simple syntax to define hierarchical structures that is widely used in computer science to define the syntax of computer languages. Don't worry, you don't have to understand EBNF as we will explain the definitions line by line.
+### Entities and Brushes
 
-    1. `Map      = Entity {Entity}`
-    2. `Entity   = {Property} {Brush}`
-    3. `Property = Key Value`
-    4. `Brush    = {Face}`
-    5. `Face     = Plane Texture Offset Scale Rotation ...`
+Maps are made of **entities**: Quake's general word for game objects, actors, or things. Players, monsters, items, doors, lights, sounds - each of these is a different type of entity you might have in your game. There are two types of entities:
+* **point entity**: a thing with an "origin" (center) at a single 3D point
```

Comment:
I think it's important to mention that entities are just a list of properties with a key and a value.